site/book-theme/header.hbs

```diff
@@ -0,0 +1,173 @@
+<style>
+.main-header {
+  /* Keep this high-z to stop fwd/prev nav from overlapping it. */
+  z-index: 102;
+  position: sticky;
+  background-image: linear-gradient(90deg, #4BC0C8 0%, #C779D0 50%, #FEAC5E 100%);
+  top: 0;
+}
```

Comment:
Great that the website header is sticky, but it covers the mdbook header when you scroll.

It's annoying to go to the top to search or use the other book controls.

Reply:
Yeah both Marno and I had a stab at fixing this, seems like it should be simple to make the mdbook header sticky but below the OT header, but we couldn't figure it out. If you want to have a go, that would be greatly appreciated!

Reply:
There is also the very-annoying related issue where hyperlinks into the book scroll the content so the top of the page (the anchor we are linking to) is also underneath the header(s). I think something like `scroll-margin-top: <height of header>` should fix it but again it didn't seem to work when I played with it.

Reply:
Yeah, I tried multiple ways of doing this. The best I was able to come up with in terms of scrolling was having the OpenTitan and mdbook headers next to each other.